lib/models/index.js

```diff
@@ -3,7 +3,7 @@ const path = require('path')
 const Sequelize = require('sequelize')
 if (!process.env.NODE_ENV) { throw new Error('NODE_ENV is not set!') }
 const dbSetting = require(path.resolve(`./settings/${process.env.NODE_ENV}.json`))
-dbSetting.define = {timestamps: false}
+dbSetting.define = {timestamps: false, underscored: true}
```

Comment:
For future associations that we introduce (if the ORM generates any table names or fields), they will be "underscored" instead of "camel-cased".